### DISCUSSION TITLE
Replybutton resize method has no effect.

### DISCUSSION THREAD

**Discussion by abduqodirovshukrullo** [def-studio/telegraph#152] [Q&A] [answered]
Reply button ->resize() method does not affect to buttons size

My code:
![image](https://user-images.githubusercontent.com/45895327/185438180-4e4d66b3-c4a9-42f1-a5e3-ddc4da8b3d23.png)

Current button:

![image](https://user-images.githubusercontent.com/45895327/185438231-e5e7a51c-0501-40ba-9ced-1dc361e37a79.png)
 
But I want smaller one
Could you give me some advise please?

**Comment by fabio-ivona** [comment, marked as answer]
hi, with the latest version it seems to work correctly:

![image](https://user-images.githubusercontent.com/8792274/186439863-b86a5194-fadf-43f8-a0c6-9d862286f384.png)

```php
 Telegraph::html('foobar')
        ->replyKeyboard(ReplyKeyboard::make()->button('foo')->requestContact()->resize(true))
        ->send();
```

Can you check against it?

**Reply by abduqodirovshukrullo** [reply]
Thank you it is working. But I want to inform that if I use ReplyButton class is instead of button() method resize() method not working. And with chunk method also has conflicts with resize.

**Reply by fabio-ivona** [reply]
@abduqodirovshukrullo thanks for the feedback, we'll check it soon

**Reply by fabio-ivona** [reply]
fixed in v1.17.1
